en/dronecan/README.md

```diff
@@ -136,6 +150,14 @@ Other Parameters:
 - If the GPS is not positioned at the vehicle centre of gravity you can account for the offset using [EKF2_GPS_POS_X](../advanced_config/parameter_reference.md#EKF2_GPS_POS_X), [EKF2_GPS_POS_Y](../advanced_config/parameter_reference.md#EKF2_GPS_POS_Y) and [EKF2_GPS_POS_Z](../advanced_config/parameter_reference.md#EKF2_GPS_POS_Z).
 - If the GPS module provides yaw information, you can enable GPS yaw fusion by setting bit 3 of [EKF2_GPS_CTRL](../advanced_config/parameter_reference.md#EKF2_GPS_CTRL) to true.
 
+#### RTK GPS
+
+Set the same parameters as for [GPS](#gps) above.
+
+Additionally set:
```

Comment:
What else should we say here that is "generic"? I.e. I know here we also set UBX mode https://docs.px4.io/main/en/dronecan/ark_rtk_gps.html#setting-up-moving-baseline-gps-heading

Essentially I'm wondering if we could get more common docs out of that above link?